packages/plugins/jwt/package.json

```
@@ -40,12 +40,17 @@
     "access": "public"
   },
   "dependencies": {
-    "jsonwebtoken": "8.5.1",
-    "jwks-rsa": "2.1.5",
-    "tslib": "^2.4.0"
+    "jsonwebtoken": "^8.5.1",
+    "jwks-rsa": "^2.1.5"
   },
   "devDependencies": {
-    "@types/jsonwebtoken": "8.5.8"
+    "@types/jsonwebtoken": "^8.5.8",
+    "graphql-scalars": "^1.22.2",
+    "tslib": "^2.4.0",
```

```suggestion
    "tslib": "^2.4.0",
```
This is imported in the compiled code so this should be under `dependencies`.

---

Don't why it moved ><' I probably messed up things here